src/io/flutter/run/SdkFields.java

```diff
@@ -25,13 +25,15 @@
  */
 public class SdkFields {
   private @Nullable String filePath;
+  private @Nullable String programArgs;
   private @Nullable String workDir;
 
-  public SdkFields() {}
+  public SdkFields() {
+  }
 
   /**
    * Creates SDK fields from a Dart file containing a main method.
-   *
+   * <p>
    * <p>(Automatically chooses the working directory.)
```

Comment:
I'm guessing the formatter wants `<p>` to be on a separate line, but it's incorrectly not deleting the `<p>` that's already there. So, if you delete the second `<p>` and reformat, I suspect it won't add it back again?